dora/core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -927,10 +929,20 @@ private int getPage(PageInfo pageInfo, int pageOffset, int bytesToRead,
         // data read from page store is inconsistent from the metastore
         LOG.error("Failed to read page {}: supposed to read {} bytes, {} bytes actually read",
             pageInfo.getPageId(), bytesToRead, ret);
+        target.offset(originOffset); //reset the offset
+        //best efforts to delete the corrupted file without acquire the write lock
+        deletePage(pageInfo, false);
```

Comment:
Do we need to delete the metadata as well?

Reply:
Deleting the metadata here will acquire more locks, which might cause dead lock.  So I just delete the page file, and page store will re-cache this page in the next visit.

Reply:
I think it's a good call, probably we can have a further improvement in the near future. But just for now, ,et's merge this pr and stop the bleeding of the data corruption issue